### PR TITLE
QUEST_LV0300 1 - 501

### PR DESCRIPTION
**Pull request by Requinn**
Identical to #1563.
Now with 100% less merge conflicts.

### CODE REVIEW

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 1]
```diff
@@ -112,8 +112,8 @@ QUEST_LV_0300_20150717_000111	Svitrigaila gave you a manual to create a magic or
 QUEST_LV_0300_20150717_000112	Destroy the Soul Compactor
 QUEST_LV_0300_20150717_000113	You have crafted Svitrigaila's Magic Orb. Now take the orb to the Soul Compactor.
 QUEST_LV_0300_20150717_000114	Destroy the Soul Compactor
-QUEST_LV_0300_20150717_000115	Destroy the Soul Compactor with Svitrigaila's orb.
-QUEST_LV_0300_20150717_000116	Once you placed Svitrigaila's orb, monsters started turning wild and rushing towards you! {nl}Fight off the monsters!
+QUEST_LV_0300_20150717_000115	Destroy the Soul Compactor with Svitrigaila's Magic orb.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 2]
```diff
@@ -112,8 +112,8 @@ QUEST_LV_0300_20150717_000111	Svitrigaila gave you a manual to create a magic or
 QUEST_LV_0300_20150717_000112	Destroy the Soul Compactor
 QUEST_LV_0300_20150717_000113	You have crafted Svitrigaila's Magic Orb. Now take the orb to the Soul Compactor.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 3]
```diff
@@ -112,8 +112,8 @@ QUEST_LV_0300_20150717_000111	Svitrigaila gave you a manual to create a magic or
...
 QUEST_LV_0300_20150717_000114	Destroy the Soul Compactor
-QUEST_LV_0300_20150717_000115	Destroy the Soul Compactor with Svitrigaila's orb.
-QUEST_LV_0300_20150717_000116	Once you placed Svitrigaila's orb, monsters started turning wild and rushing towards you! {nl}Fight off the monsters!
+QUEST_LV_0300_20150717_000115	Destroy the Soul Compactor with Svitrigaila's Magic orb.
+QUEST_LV_0300_20150717_000116	Once you placed Svitrigaila's Magic Orb, monsters started turning wild and rushing towards you! {nl}Fight off the monsters!
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 4]
```diff
@@ -393,28 +393,28 @@ QUEST_LV_0300_20151001_000392	You are now ready to fix the detector. Talk with K
...
-QUEST_LV_0300_20151001_000411	Talk with the Cleric Master on how to obtain the crystal that is needed by Svitrigaila
-QUEST_LV_0300_20151001_000412	Return to demon Svitrigaila
-QUEST_LV_0300_20151001_000413	The Cleric Master has told you that he can't help the demons. It is unfortunate, but please tell this to the demon, Svitrigaila.
-QUEST_LV_0300_20151001_000414	It seems that the poisonous plant isn't effective on demon Svitrigaila. Talk to him again.
-QUEST_LV_0300_20151001_000415	Demon Svitrigaila wants to die using the crystal since the poisonous plant isn't effective. Meet with the Cleric Master and obtain the divine water.
+QUEST_LV_0300_20151001_000404	Your stamina keeps decreasing as if you are losing your spirit. Quickly find someone who can fix this situation.
+QUEST_LV_0300_20151001_000405	Talk with Demon Svitrigaila
+QUEST_LV_0300_20151001_000406	Demon Svitrigaila is muttering something that that is difficult to understand. I hope you somehow heard his story.
+QUEST_LV_0300_20151001_000407	Collect some poisonous plants growing in the Zinios Survey Office
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Remove 'some' 
poisonous plants -> Poisonous Plant

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 5]
```diff
@@ -393,28 +393,28 @@ QUEST_LV_0300_20151001_000392	You are now ready to fix the detector. Talk with K
...
+QUEST_LV_0300_20151001_000408	Demon Svitrigaila told you that he will remove the curse from you if you help him die. Obtain some poisonous plants from the Zinios Survey Office.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: poisonous plants -> Poisonous Plant

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 6]
```diff
@@ -393,28 +393,28 @@ QUEST_LV_0300_20151001_000392	You are now ready to fix the detector. Talk with K
...
+QUEST_LV_0300_20151001_000409	Hand over the poisonous plants to Demon Svitrigaila
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: poisonous plants -> Poisonous Plant

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 7]
```diff
@@ -393,28 +393,28 @@ QUEST_LV_0300_20151001_000392	You are now ready to fix the detector. Talk with K
...
+QUEST_LV_0300_20151001_000410	It seems like you've obtained all the poisonous plants Demon Svitrigaila wanted. Hand them over to Svitrigaila.
+QUEST_LV_0300_20151001_000411	Talk with the Cleric Master on how to obtain the crystal that Svitrigaila needs
+QUEST_LV_0300_20151001_000412	Return to Demon Svitrigaila
+QUEST_LV_0300_20151001_000413	The Cleric Master has told you that he can't help the demons. It is unfortunate, but please tell this to Demon Svitrigaila.
+QUEST_LV_0300_20151001_000414	It seems that the poisonous plant isn't effective on Demon Svitrigaila. Talk to him again.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: poisonous plants -> Poisonous Plant

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 8]
```diff
@@ -424,25 +424,25 @@ QUEST_LV_0300_20151001_000423	Talk with Svitrigaila what you should do to destro
...
-QUEST_LV_0300_20151001_000431	Hand over the core of the barrier stone to demon Svitrigaila
+QUEST_LV_0300_20151001_000431	Hand over the core of the barrier stone to Demon Svitrigaila
 QUEST_LV_0300_20151001_000432	You've destroyed all the barrier stones and obtained the cores. Hand over the cores of the barrier stones to Svitrigaila.
-QUEST_LV_0300_20151001_000433	You've destroyed all the barrier stones and obtained the cores so talk with demon Svitrigaila about what you should do next.
-QUEST_LV_0300_20151001_000434	Craft the orbs of Svitrigaila
-QUEST_LV_0300_20151001_000435	Demon Svitrigaila has asked you to create the orbs that would destroy the integrator. Obtain detonators from the petrified monsters and use the craft manuals to create the orbs of Svitrigaila.
+QUEST_LV_0300_20151001_000433	You've destroyed all the barrier stones and obtained the cores so talk with Demon Svitrigaila about what you should do next.
+QUEST_LV_0300_20151001_000434	Craft Svitrigaila's Magic Orb
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic Orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 9]
```diff
@@ -424,25 +424,25 @@ QUEST_LV_0300_20151001_000423	Talk with Svitrigaila what you should do to destro
...
+QUEST_LV_0300_20151001_000435	Demon Svitrigaila has asked you to create the orbs that would destroy the integrator. Obtain detonators from the petrified monsters and use the craft manuals to create Svitrigaila's Magic Orbs.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic Orbs -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 10]
```diff
@@ -424,25 +424,25 @@ QUEST_LV_0300_20151001_000423	Talk with Svitrigaila what you should do to destro
...
 QUEST_LV_0300_20151001_000436	Destroy the integrator at the collapsed temple ground
-QUEST_LV_0300_20151001_000437	You've created the orb of Svitrigaila. Take this orb to the integrator.
+QUEST_LV_0300_20151001_000437	You've created Svitrigaila's Magic Orb. Take this orb to the integrator.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic Orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 11]
```diff
@@ -424,25 +424,25 @@ QUEST_LV_0300_20151001_000423	Talk with Svitrigaila what you should do to destro
...
 QUEST_LV_0300_20151001_000438	Destroy the integrator
-QUEST_LV_0300_20151001_000439	The time to destroy the integrator has come. Take the orb of Svitrigaila and destroy the integrator.
+QUEST_LV_0300_20151001_000439	The time to destroy the integrator has come. Take Svitrigaila's Magic Orb and destroy the integrator.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Svitrigaila's Magic Orb -> Svitrigaila's Orb

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 12]
```diff
@@ -424,25 +424,25 @@ QUEST_LV_0300_20151001_000423	Talk with Svitrigaila what you should do to destro
...
 QUEST_LV_0300_20151001_000440	As Svitrigaila puts down the orb, the monsters who felt the power are gathering near it. Defeat the monsters.
 QUEST_LV_0300_20151001_000441	The integrator has been destroyed. Return to Svitrigaila.
-QUEST_LV_0300_20151001_000442	Listen to the last words of demon Svitrigaila
-QUEST_LV_0300_20151001_000443	Now is the time to listen to the wishes of demon Svitrigaila. Listen to his words.
+QUEST_LV_0300_20151001_000442	Listen to the last words of Demon Svitrigaila
+QUEST_LV_0300_20151001_000443	Now is the time to listen to the wishes of Demon Svitrigaila. Listen to his words.
 QUEST_LV_0300_20151001_000444	You've pulled out the poisonous plant!
-QUEST_LV_0300_20151001_000445	Collect the last breathes
+QUEST_LV_0300_20151001_000445	Collect the last breaths
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: breaths -> breath

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 13]
```diff
@@ -405,14 +405,14 @@ QUEST_LV_0300_20151001_000404	Your stamina keeps decreasing as if you are losing
 QUEST_LV_0300_20151001_000405	Talk with Demon Svitrigaila
 QUEST_LV_0300_20151001_000406	Demon Svitrigaila is muttering something that that is difficult to understand. I hope you somehow heard his story.
 QUEST_LV_0300_20151001_000407	Collect some poisonous plants growing in the Zinios Survey Office
-QUEST_LV_0300_20151001_000408	Demon Svitrigaila told you that he will remove the curse from you if you help him die. Obtain some poisonous plants from the Zinios Survey Office.
-QUEST_LV_0300_20151001_000409	Hand over the poisonous plants to Demon Svitrigaila
-QUEST_LV_0300_20151001_000410	It seems like you've obtained all the poisonous plants Demon Svitrigaila wanted. Hand them over to Svitrigaila.
+QUEST_LV_0300_20151001_000408	Demon Svitrigaila told you that he will remove the curse from you if you help him die. Obtain Poisonous Plant from the Zinios Survey Office.
+QUEST_LV_0300_20151001_000409	Hand over the Poisonous Plant to Demon Svitrigaila
+QUEST_LV_0300_20151001_000410	It seems like you've obtained all the Poisonous Plant Demon Svitrigaila wanted. Hand them over to Svitrigaila.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv:410]: Poisonous Plants

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 14]
```diff
@@ -405,14 +405,14 @@ QUEST_LV_0300_20151001_000404	Your stamina keeps decreasing as if you are losing
 QUEST_LV_0300_20151001_000405	Talk with Demon Svitrigaila
 QUEST_LV_0300_20151001_000406	Demon Svitrigaila is muttering something that that is difficult to understand. I hope you somehow heard his story.
 QUEST_LV_0300_20151001_000407	Collect some poisonous plants growing in the Zinios Survey Office
-QUEST_LV_0300_20151001_000408	Demon Svitrigaila told you that he will remove the curse from you if you help him die. Obtain some poisonous plants from the Zinios Survey Office.
-QUEST_LV_0300_20151001_000409	Hand over the poisonous plants to Demon Svitrigaila
-QUEST_LV_0300_20151001_000410	It seems like you've obtained all the poisonous plants Demon Svitrigaila wanted. Hand them over to Svitrigaila.
+QUEST_LV_0300_20151001_000408	Demon Svitrigaila told you that he will remove the curse from you if you help him die. Obtain Poisonous Plant from the Zinios Survey Office.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv:408]: Obtain the Poisonous Plant

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 15]
```diff
@@ -107,13 +107,13 @@ QUEST_LV_0300_20150717_000106	Destroy the Terminal
 QUEST_LV_0300_20150717_000107	Svitrigaila says that the third terminal will get knocked out if anything comes near it. Retrieve the core from the petrified monsters nearby and throw it at the terminal to destroy it.
 QUEST_LV_0300_20150717_000108	And Eternal Repose
 QUEST_LV_0300_20150717_000109	Now that you've gotten rid of all the terminals, return to Svitrigaila.
-QUEST_LV_0300_20150717_000110	Find materials to craft the Magic Orb
-QUEST_LV_0300_20150717_000111	Svitrigaila gave you a manual to create a magic orb that can free the trapped souls from the Soul Compactor. Gather the necessary crafting materials that Svitrigaila was unable to get from monsters and then use the manual to craft Svitrigaila's Magic Orb.
+QUEST_LV_0300_20150717_000110	Find materials to craft the Orb
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Find materials to craft Svitrigaila's Orb

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**